setup.cfg

```diff
@@ -18,14 +18,15 @@ classifiers =
     Programming Language :: Python :: 3.7
 
 [options]
+zip_safe = True
```

Comment:
This would make it easier for dependants the installation via wheels (https://setuptools.readthedocs.io/en/latest/setuptools.html#setting-the-zip-safe-flag).